zenoh/src/api/time.rs

```diff
@@ -15,6 +15,7 @@ use std::time::{SystemTime, UNIX_EPOCH};
 
 use zenoh_protocol::core::{Timestamp, TimestampId};
 
+// TODO: Shall we remove this new_timestamp in favoir of the src/api/session::Session::new_timestamp();
```

Comment:
Yes, there was discussion about it, created issue https://github.com/eclipse-zenoh/zenoh/issues/1179